plugins/SitesManager/templates/_displayJavascriptCode.twig

```diff
@@ -31,6 +31,11 @@
     <p>{{ 'CoreAdminHome_HttpTrackingApiDescription'|translate('<a href="https://developer.matomo.org/api-reference/tracking-api" rel="noreferrer noopener" target="_blank">','</a>')|raw }}</p>
     <p></p>
 
-    {{ postEvent('Template.endTrackingHelpPage') }}
+    {% set emailSubject = 'Matomo' ~ 'General_JsTrackingTag'|translate %}
+    <a class="btn"
+       href="mailto:?subject={{ emailSubject|url_encode }}&body={{ emailBody|replace({">\n<": "><"})|url_encode }}">
```

Comment:
haven't really looked at the PR yet. Would it be possible to not use this replace here and instead directly remove the HTML in `_trackingCodeEmail.twig`?

Reply:
It's stripping out the newlines between tags - I could minify the template instead but I think that would make it very hard to read.

Reply:
What I meant is it's not possible to add HTML tags in a mailto link anyway? And the line breaks can be taken over from the twig template?